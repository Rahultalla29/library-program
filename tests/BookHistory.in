ADD BOOK sample1.csv 111111
ADD BOOK sample1.csv 111112
ADD MEMBER Jeff
ADD MEMBER James Donald
BOOK HISTORY 111111
BOOK HISTORY 111112
RENT 100000 111111
RENT 100000 111112
RELINQUISH 100000 111112
RELINQUISH 100000 111111
RENT 100001 111111
RELINQUISH 100001 111111
BOOK HISTORY 111111
BOOK HISTORY 111112
EXIT
