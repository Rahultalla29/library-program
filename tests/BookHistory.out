user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: Successfully added: The Hobbit (J.R.R. Tolkien).

user: Success.

user: Success.

user: No rental history.

user: No rental history.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: 100000
100001

user: 100000

user: Ending Library process.
